src/components/get-help-modal/GetHelpModal.tsx

```diff
@@ -0,0 +1,21 @@
+import { useLocales } from 'src/locales';
+import FavoriteIcon from 'src/assets/icons/FavoriteIcon';
+
+import { Container, IconContainer, SubTitle, Text, Link } from './styles';
+
+export default function GetHelpModal(): JSX.Element {
+  const { translate } = useLocales();
+
+  return (
+    <Container>
+      <IconContainer>
+        <FavoriteIcon />
+      </IconContainer>
+      <SubTitle>{translate('getHelpModal.subtitle')}</SubTitle>
+      <Text>{translate('getHelpModal.text')}</Text>
+      <Link href={`mailto:${translate('needHelpModal.email')}`}>
```

Comment:
not sure if the link to email should be in locales, I think there will be one support email, maybe it would be better to store it in constants?

Reply:
fixed